cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -591,10 +581,20 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             return this.model.get('ancestor_has_staff_lock');
         },
 
+        getContentGroups: function() {
```

Comment:
Do these methods need to be in the AbstractVisibilityEditor? In master, that base class is used for StaffLockEditor and ContentVisibilityEditor. I don't think either of those need content group or group access information. See ContentVisibilityEditor for an example of how getContext can be extended in UnitAccessEditor to provide the content group and access information for that specific editor.